validator/validator.protoascii

```diff
@@ -2278,6 +2278,23 @@ tags: {
     supported_layouts: NODISPLAY
   }
 }
+# <amp-embed>
```

Comment:
@powdercloud since i will be removing shortly this from this PR as per your instructions, i think this comment above should be directed to you. 
should add a comment that changes on `amp-ad` should be done on `amp-embed` and vice versa.